### DISCUSSION TITLE
Filament RichEditor

### DISCUSSION THREAD

**Discussion by lloricode** [UniSharp/laravel-filemanager#1163] [General]
Hi i am convert project to https://filamentphp.com/, is there way to use this in [RichEditor field](https://filamentphp.com/docs/2.x/forms/fields#rich-editor)?

**Comment by streamtw** [comment]
I am sorry, we currently do not have a integration option for filamentphp. If it is heavily used by Laravel users, I might consider implementing a new integration option for filamentphp.
